tests/test_cli.py

```diff
@@ -21,7 +22,7 @@ def runner():
             yield r
     finally:
         # Make sure other tests are not affected by the ones with --verbose
-        LOG.setLevel(logging.INFO)
+        importlib.reload(sys.modules['logging'])
```

Comment:
OK, could you send your PR to my branch: https://github.com/junaruga/rpm-list-builder/tree/feature/nit-picking-make-init.py-empty ?
Because I tried your way, and other ways. But still I can not pass unit tests on those ways.